addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -168,110 +168,109 @@
             </t>
 
             <div class="o_wslides_course_main">
-                <!-- ========== TRAINING COURSE ========== -->
-                <div t-if="channel.channel_type == 'training'" class="container">
-                    <div class="row">
-                        <!-- Training Sidebar -->
-                        <div class="col-12 col-md-4 col-lg-3 mt-3 mt-md-0">
-                            <t t-call="website_slides.course_sidebar"/>
-                        </div>
+                <div class="container mb-5">
 
-                        <!-- Training Content -->
-                        <div class="col-12 col-md-8 col-lg-9">
-                            <ul class="nav nav-tabs o_wslides_nav_tabs flex-nowrap" role="tablist" id="profile_extra_info_tablist">
+                    <!-- Row top tabs -->
+                    <div class="row">
+                        <div class="col-12 col-md-8 offset-md-4 offset-lg-3 col-lg-9">
+                            <ul class="nav nav-tabs o_wslides_nav_tabs o_wslides_doc_nav_tabs flex-nowrap" role="tablist" id="profile_extra_info_tablist">
```

Comment:
shouldn't we add `o_wslides_doc_nav_tabs` only for documentation type only ? 
you could use t-att-class